client/src/main/java/tds/assessment/Segment.java

```diff
@@ -143,7 +168,7 @@ public void setFieldTestMinItems(int fieldTestMinItems) {
     /**
      * @return the maximum number of field test items in the {@link tds.assessment.Segment}
      */
-    public int getFieldTestMaxItems() {
+    public Integer getFieldTestMaxItems() {
```

Comment:
In the case of fieldTestStartPosition/EndPosition, I suppose although they are typically null in the `tblsetofadminsubjects` table, effectively it doesn't make much of a difference if they are null or zeros - in the case that there are ft min/maxes defined, we expect the start and end position to be non-null and non-zero, both would be Illegal States. And if there aren't any ft min/maxes defined, then we really don't care whats in the other field test columns. I think it's fine to make these primitives as well.